intern/service/boss/baseversion.py

```diff
@@ -209,6 +209,39 @@ def build_cutout_url(
                 urlWithParams += '{}={}'.format(k,v)
         return urlWithParams
 
+    def build_cutout_to_black_url(
+        self, resource, url_prefix, resolution, x_range, y_range, z_range, time_range=None):
+        """Build the url to access the cutout_to_black function of the Boss' volume service.
+        TODO: Maybe delete this function and add a base_url parameter to build_cutout_url?
+        Args:
+            url_prefix (string): Do not end with a slash.  Example of expected value: https://api.theboss.io
+            resolution (int): 0 indicates native resolution.
+            x_range (list[int]): x range such as [10, 20] which means x>=10 and x<20.
+            y_range (list[int]): y range such as [10, 20] which means y>=10 and y<20.
+            z_range (list[int]): z range such as [10, 20] which means z>=10 and z<20.
+            time_range (optional [list[int]]): time range such as [30, 40] which means t>=30 and t<40.
+
+        Returns:
+            (string): Full URL to access API.
+
+        Raises:
+            (RuntimeError): if *_range invalid.
+        """
+        baseUrl = self.build_url(resource, url_prefix, 'cutout/to_black', req_type='cutout')
```

Comment:
these vars (`base_url`, `url_with_params`) should all be `underscore_vars` instead of `camelCase`